.github/workflows/changelog.yml

```diff
@@ -5,27 +5,10 @@ on:
     types: [opened, synchronize, edited, reopened]
 
 jobs:
-  check-description:
+  check-pr-fields:
```

Comment:
I know it's technically no longer correct - but easier to leave this as `check-description` so that the required check value stays the same in the repository settings.

And _technically_ we are still checking the PR description here, just in more detail than before.

Reply:
Ok ....i agree then